apps/emqx_mq/src/emqx_mq_gc/emqx_mq_gc_worker.erl

```diff
@@ -0,0 +1,161 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2025 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_mq_gc_worker).
+
+-moduledoc "The module is responsible for garbage collection of Message "
+"Queue data.".
```

Comment:
Nit:
```suggestion
-moduledoc """"
The module is responsible for garbage collection of Message
Queue data.
""".
```

Reply:
Also please describe how the GC logic works, it's not super obvious from the code.